.github/workflows/ci-release-gate.yml

```
@@ -56,528 +45,168 @@ jobs:
-            --predicate "sbom-${{ matrix.name }}.spdx.json" \
-            --type "${{ env.SBOM_PREDICATE_TYPE }}" \
-            "$IMAGE"
-
-      - name: Attest Build Provenance
-        uses: actions/attest-build-provenance@a2bbfa25375fe432b6a289bc6b6cd05ecd0c4c32 #v4.1.0
-        with:
-          subject-name: docker.io/${{ secrets.DOCKER_HUB_USERNAME }}/${{ matrix.image_repo }}
-          subject-digest: ${{ env.DIGEST }}
-          push-to-registry: true
+      - name: Generate SBOM and Attest - GHCR
+        uses: ./.github/actions/sbom-and-attest
```

actionlint (1.7.12)

[error] missing input "registry_username" which is required by action "Generate SBOM and Attest" defined at "./.github/actions/sbom-and-attest". all required inputs are "image_digest", "image_name", "registry", "registry_password", "registry_username", "sbom-predicate-type", "sbom-repository"

(action)